### ISSUE TITLE
[Issue]: Crash entering Nether

### ISSUE THREAD

**Issue by SerUameN** [The-Nexus-Project/Limitless-7#3]
### What happened?

We're getting kicked off our minecraft Server when trying to enter the Nether Dimension. When trying to rejoin the error persists and we get immediately kicked again. Only using the tp Command in the consol in the short window between Player joined and Player left helped so we could keep playing. 

### Pack Version

Version: LL7-0.11.0
Forge Version: 47.1.3

### To Reproduce

1.join Nether

### Are you using OptiFine or Oculus?

- [ ] Using OptiFine
- [X] Using Oculus
- [ ] No. None of them. Unmodified.

### Are you on a Server?

- [X] Yes.
- [ ] No.

### Relevant log output / crash report.

```shell
[22:13:32] [Netty Client IO #1/ERROR]: Exception caught in connection handler!
 java.lang.IndexOutOfBoundsException: readerIndex(14) + length(4) exceeds writerIndex(16): PooledUnsafeDirectByteBuf(ridx: 14, widx: 16, cap: 16)
	at MC-BOOTSTRAP/io.netty.buffer@4.1.82.Final/io.netty.buffer.AbstractByteBuf.checkReadableBytes0(AbstractByteBuf.java:1442)
	at MC-BOOTSTRAP/io.netty.buffer@4.1.82.Final/io.netty.buffer.AbstractByteBuf.readInt(AbstractByteBuf.java:809)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.network.FriendlyByteBuf.readInt(FriendlyByteBuf.java:1237)
	at TRANSFORMER/irons_spellbooks@1.20.1-1.1.8/io.redspace.ironsspellbooks.entity.mobs.MobSyncedCastingData$1.read(MobSyncedCastingData.java:28)
	at TRANSFORMER/irons_spellbooks@1.20.1-1.1.8/io.redspace.ironsspellbooks.entity.mobs.MobSyncedCastingData$1.m_6709_(MobSyncedCastingData.java:15)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.network.syncher.SynchedEntityData$DataValue.m_252951_(SynchedEntityData.java:284)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.network.syncher.SynchedEntityData$DataValue.m_252860_(SynchedEntityData.java:279)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.network.protocol.game.ClientboundSetEntityDataPacket.m_252908_(ClientboundSetEntityDataPacket.java:32)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.network.protocol.game.ClientboundSetEntityDataPacket.<init>(ClientboundSetEntityDataPacket.java:16)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.network.ConnectionProtocol$PacketSet.m_178327_(ConnectionProtocol.java:459)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.network.ConnectionProtocol.m_178321_(ConnectionProtocol.java:522)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.network.PacketDecoder.decode(PacketDecoder.java:32)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:519)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:458)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:280)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:336)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:308)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:336)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:323)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:444)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:280)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at MC-BOOTSTRAP/io.netty.codec@4.1.82.Final/io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at MC-BOOTSTRAP/io.netty.handler@4.1.82.Final/io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at MC-BOOTSTRAP/io.netty.transport@4.1.82.Final/io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at MC-BOOTSTRAP/io.netty.common@4.1.82.Final/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at MC-BOOTSTRAP/io.netty.common@4.1.82.Final/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base/java.lang.Thread.run(Thread.java:833)

[22:13:32] [Netty Client IO #1/WARN]: Packet not registered: ClientboundDisconnectPacket
[22:13:32] [Netty Client IO #1/WARN]: Packet:ClientboundDisconnectPacket 
[22:13:32] [Netty Client IO #1/WARN]: Packet data:
{
  "f_132075_": {
    "f_237194_": {
      "f_237497_": "disconnect.genericReason",
      "f_237498_": [
        "Internal Exception: io.netty.handler.codec.DecoderException: java.lang.IndexOutOfBoundsException: readerIndex(14) + length(4) exceeds writerIndex(16): PooledUnsafeDirectByteBuf(ridx: 14, widx: 16, cap: 16)"
      ],
      "f_237500_": []
    },
    "f_237195_": [],
    "f_237196_": {},
    "f_237197_": {}
  }
}
[22:13:32] [Render thread/INFO]: [EMI] Disconnecting from server, EMI data cleared
[22:13:32] [Thread-79/INFO]: [EMI] Clearing data
[22:13:32] [Render thread/INFO]: Released client Flux Networks cache
[22:13:32] [Render thread/INFO]: JEI StartEventObserver received class net.minecraftforge.client.event.ClientPlayerNetworkEvent$LoggingOut
[22:13:32] [Render thread/INFO]: JEI StartEventObserver transitioning state from JEI_STARTED to DISABLED
[22:13:32] [Render thread/INFO]: Stopping JEI
[22:13:32] [Render thread/INFO]: Sending Runtime Unavailable...
[22:13:32] [Render thread/INFO]: Stopping JEI GUI
[22:13:32] [Render thread/INFO]: Sending Runtime Unavailable took 2.079 ms
[22:13:32] [Render thread/ERROR]: Something went wrong. Could not find JEI Search Text Field object
[22:13:32] [Render thread/ERROR]: Something went wrong. Could not find JEI Search Text Field object
[22:13:32] [Render thread/ERROR]: Something went wrong. Could not find JEI Search Text Field object
[22:13:32] [Render thread/INFO]: Stopping worker threads
[22:13:32] [Render thread/ERROR]: Something went wrong. Could not find JEI Search Text Field object
[22:13:32] [Render thread/INFO]: Reset 1 configs to their client-side state.
```

**Comment by Gbergz** [comment]
Known issue. No eta on a fix for now. Teleport yourself away with an NBT editor.

**Comment by Gbergz** [comment]
"Fixed" in v0.12.0.
